### ISSUE TITLE
question about your modification

### ISSUE THREAD

**Issue by zkself** [poojahira/image-captioning-bottom-up-top-down#7]
I am also trying to implement this method, my code is almost same as your code except the Discriminative supervision. could you please tell me your motivation about this modification?

**Comment by poojahira** [comment]
It's just an additional loss function, for discriminative loss. Totally optional. I got the idea from a few papers that are using this for image captioning, in addition to cross entropy loss. I thought it might improve performance. If you're interested I can post the links of these papers.

**Comment by zkself** [comment]
@poojahira   that's will be nice !  if you can offer me those papers. 
thanks a lot!!!!!

**Comment by poojahira** [comment]
These are two that I went through that used discriminative supervision: 
https://arxiv.org/abs/1804.00887
https://arxiv.org/abs/1605.07912
There might be others as well.
